lib/pricefeed.ts

```diff
@@ -78,7 +78,7 @@ export default class PriceFeed {
     this.logger.debug(`Got PYTH price for %s: %d`, assetId, pythPrice);
     const dropExchangeRateResult = await queryContractOnHeight<string>(
       client,
-      this.params.assets[assetId.split('_')[0]].core_contract,
+      this.params.assets[assetId].core_contract,
```

Comment:
how do we handle `dATOM_NTRN`-like assets now?

Reply:
I moved this split https://github.com/hadronlabs-org/drop-points-crawler/blob/96913ef5e39d2f9d9b2c334c534684d39c52ae7a/crawler/index.ts#L148 to this line, didn't want to shadow this split inside price feed method